plato/draw/Arrows2D.py

```diff
@@ -20,7 +19,7 @@ class Arrows2D(Polygons):
     """
 
     def __init__(self, *args, **kwargs):
-        super(Polygons, self).__init__(*args, **kwargs)
+        super().__init__(*args, **kwargs)
```

Comment:
This change is to fix the MRO of the `Arrows2D` subclass in `fresnel` and potentially other backends.

The MRO for `plato.draw.fresnel.Arrows2D` looks like this:
```python
>>> for cls in plato.draw.fresnel.Arrows2D.mro(): print(cls)
<class 'plato.draw.fresnel.Arrows2D.Arrows2D'>
<class 'plato.draw.Arrows2D.Arrows2D'>
<class 'plato.draw.fresnel.Polygons.Polygons'>
<class 'plato.draw.fresnel.FresnelPrimitive.FresnelPrimitiveSolid'>
<class 'plato.draw.fresnel.FresnelPrimitive.FresnelPrimitive'>
<class 'plato.draw.Polygons.Polygons'>
<class 'plato.draw.internal.Shape'>
<class 'object'>
```

By doing this, we just call `__init__` on the next higher class in the MRO, after this one. If we use `super(Polygons, self)`, we call `__init__` from `plato.draw.Polygons.Polygons` instead of `plato.draw.fresnel.Polygons.Polygons` as we desired.

Reply:
Previously, this only worked because the `fresnel` backend wasn't using inheritance well -- in my understanding, this method (in addition to the mixin class for `FresnelPrimitive`) is the "less hacky" way to do this.